### ISSUE TITLE
Remove duplicate images from collected data

### ISSUE THREAD

**Issue by James-Leslie** [datahounds/what-whale-classifier#13]
Initial validation accuracy has me worried about duplicate images in the training and validation sets. It's a bit tricky to simply eyeball it, but found a [cool tutorial](https://www.pyimagesearch.com/2020/04/20/detect-and-remove-duplicate-images-from-a-dataset-for-deep-learning/) for removing duplicates from a dataset.

My initial thoughts are that we should set this up as a notebook which taps into our existing data file structure and removes duplicates in place (i.e. parse the files inside `data/` and all of its subdirectories).
